src/components/Time/index.tsx

```diff
@@ -13,8 +13,8 @@ var event = new Event('timestamp')
 let minute: number
 
 setInterval(() => {
-  const date: Date = new Date()
-  const currentMinutes: number = date.getMinutes()
+  var date: Date = new Date()
+  var currentMinutes: number = date.getMinutes()
```

Comment:
Any reason you changed these from `const` to `var`? It's normally preferred to use `const` unless you need to change the variable assignment then `let` is more preferred. I don't remember last time I've actually used `var` :)